src/js/actions/history.js

```diff
@@ -104,10 +105,13 @@ define(function (require, exports) {
                 });
             })
             .then(function () {
-                var currentDocument = this.flux.store("application").getCurrentDocument();
+                var document = this.flux.store("application").getCurrentDocument(),
```

Comment:
Above in the `catch` clause it seems to imply that it's a possibility that no document will currently be open. I realize it's comparing apples to oranges somewhat, but is that something we should test for here? Should this bail out early if `getCurrentDocument()` returns null? Or is there some reason why you trust that that can never happen, despite the comment above?